### PR TITLE
Added top button to index.html and explore.html.

### LINKED ISSUE #63: Top button for the home and explore page.
**Is your feature request related to a problem? Please describe.**
The options to sign up, logout, explore or other such options are available on the top of the home page and explore page, but when you are scrolling down and you want to use one of these options you have to scroll all the way to the top of the page Which is tidious.

**Describe the solution you'd like**
A very good solution to this problem will be a top button on the bottom right corners of the home and explore page. This button can have an upward arrow or top written on it and it will take you to the top of the page when clicked. The button can be customised and made attractive using CSS.

**Are you willing to contribute to this issue?** 
Yes, I am a participant in GsSOC'21.

**Describe alternatives you've considered**
I thought of using a sticky navigation bar but personally I don't like that kind of navigation bar, it covers the contents and is irritating.

### PR DESCRIPTION
**Pull request by kanishka8276**
## Description
- I  have added a top button to index.html and explore.html. It is in the bottom right corner and will take the user to the top op the page.
- Resolves #63 

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/Praful932/Kitabe/blob/master/CONTRIBUTING.md).
- [x] My changes do not edit/add any unrequired files.
- [x] My changes are covered by tests.
Top button
Screenshot (56)
Top button when hover
Screenshot (57)

### CODE REVIEW

**Comment by maneprajakta** [comment]
The issue seems valid to me.
@kanishka8276 why did you closed the issue 🙂 ? If you don't want to work on this let us know. We will assign someone else.

**Comment by kanishka8276** [comment]
Sorry but I didn't closed the issue. I closed the PR . Because it was blocked from merging. So, I thought of making the required changes and then make another PR. I am still working on it.

**Comment by Praful932** [comment]
Hi @kanishka8276, it's fine you can unstage that file without closing PR, follow steps [here](https://stackoverflow.com/questions/38743912/remove-a-file-from-a-git-pull-request). Also Update the PR description with relevant details & GIFs

**Comment by maneprajakta** [comment]
> Sorry @maneprajakta but I didn't closed the issue. I closed the PR . Because it was blocked from merging as some checks were not passed. So, I thought of making the required changes and then make another PR. I am still working on it.

No problem 

> Sorry @maneprajakta but I didn't closed the issue. I closed the PR . Because it was blocked from merging as some checks were not passed. So, I thought of making the required changes and then make another PR. I am still working on it.

No issue you need to remove localhost from allowed host and also secret key it was before then PR will pass all checks.
You can make a fresh branch and do changes to it and make another PR :rocket: .

**Comment by kanishka8276** [comment]
Ok! Thank you.

**Comment by kanishka8276** [comment]
@Praful932  and @maneprajakta  please review this PR. I have made all the required changes. Sorry,I couldn't unstage settings.py using the command you suggested. It wasn't working for me.

**Comment by Praful932** [comment]
> Update the PR description with relevant details & GIFs

@kanishka8276 
Btw the `settings.py` got unstaged :)

**Comment by kanishka8276** [comment]
> > Update the PR description with relevant details & GIFs
> 
> @kanishka8276
> Btw the `settings.py` got unstaged :)

Oh! . The PR description contains the current images of the top button. And after that I just added html scrollbar behaviour property. Rest is the Same. What changes shall I make in the PR comment?

**Comment by Praful932** [comment]
> > > Update the PR description with relevant details & GIFs
> > 
> > 
> > @kanishka8276
> > Btw the `settings.py` got unstaged :)
> 
> Oh! . The PR description contains the current images of the top button. And after that I just added html scrollbar behaviour property. Rest is the Same. What changes shall I make in the PR comment?

Record GIFs and update it in the description

**Comment by kanishka8276** [comment]
@maneprajakta @Praful932 I will have to make changes in this. Actually previously I tested the code for the button only by creating a testing   HTML file because I was unable to setup the project. Now I have set it up and found that  the top button is not visible.
I have tried increasing the z-index and changing html.onscroll to body.onscroll but still it is not visible.
Can you please suggest me some changes.
I am trying my best and will try to do it as soon as possible. Sorry, for the delay.

File: templates/mainapp/explore.html [hunk 1]
```diff
@@ -12,7 +12,24 @@
 
 
 {% block main %}
-
+<!-----------------------------------------------------------Top Button------------------------------------------------------------------------->
+	<script>
```
Comment (maneprajakta) [inline templates/mainapp/explore.html:16]: You can add this script here in [*layout.html*](https://github.com/Praful932/Kitabe/blob/91c168d1bcd5888c835fc48bf3ff3e0ac05064b7/templates/mainapp/layout.html#L357) this reduces repetition of code.

File: templates/mainapp/index.html [hunk 2]
```diff
@@ -38,7 +38,23 @@
     <!-- e carousel -->
 </section>
 <!-- e section -->
-
+/* -----------------------------------------------------------Top Button-------------------------------------------------------------------------*/
+	<script>
```
Comment (maneprajakta) [inline templates/mainapp/index.html:42]: If you put the script  in layout.html there will be no need for this

**Review by maneprajakta — CHANGES_REQUESTED** [review]
@kanishka8276 Take your time, go through templates to see how they are inherited. Avoid repetition as much as possible. Also make sure you five appropriate id to button and names to function.

**Comment by kanishka8276** [comment]
@maneprajakta ok! I Will do that. Thank you for the help!😀

**Comment by Praful932** [comment]
@kanishka8276 I have only seen the top button you attached in the image, would suggest to keep it consistent with the project theme

**Comment by kanishka8276** [comment]
Ok!

**Comment by kanishka8276** [comment]
@Praful932 and @maneprajakta  .
Please review PR #88
I have tried to make the top button according to the theme of the page but if you want any colour changes please suggest

**Pull request closed without merging**